modeltranslation/utils.py

```diff
@@ -112,7 +117,9 @@ def unique(seq):
     return (x for x in seq if x not in seen and not seen.add(x))
```

Comment:
mypy complains with:

`modeltranslation/utils.py:119: error: "add" of "set" does not return a value (it only ever returns None)  [func-returns-value]`

this implementation seems to be a hacky way of adding the element in the set while still keeping it a one liner. Maybe a more explicit implementation should be used instead?

Reply:
For now, add `# type: ignore` for this line

And we can change implementation to non-hacky later.